Tests/Editor/CoreApi/GameLiftComputeManagerTests.cs

```diff
@@ -84,7 +84,7 @@ public void RegisterFleetCompute_WhenCorrectInputs_ExpectSuccess()
             _gameLiftWrapperMock.Verify(wrapper => wrapper.RegisterCompute(It.IsAny<RegisterComputeRequest>()), Times.Once);
             _coreApiMock.Verify(f => f.PutSetting(It.IsAny<string>(), It.IsAny<string>()), Times.Exactly(3));
             
-            Assert.IsTrue(registerComputeResponse.Success);
+            Assert.IsTrue(registerComputeResponse);
```

Comment:
Is this right? Seems like the Response Success/Failure is what needs to be asserted rather than the Response itself

Reply:
Same for other changes in this file.

Reply:
Fixed, will be broken until #111 Is implemented as it turns them back into responses.